salt/metalk8s/salt/master/files/master-99-metalk8s.conf.j2

```diff
@@ -37,4 +37,7 @@ external_auth:
       - '@jobs'
     storage-operator:
       - '*':
-        - 'test.ping'
+        - 'test.rand_sleep'
+        - 'disk.dump'
+        - 'state.sls'
```

Comment:
We can further limit this, please file a ticket.

Reply:
Like that:

```yaml
        - 'test.rand_sleep'
        - 'disk.dump'
        - 'state.sls':
            args:
              - 'metalk8s.volumes'
```
?

Reply:
Something along those lines. Not required in this PR as far as I'm concerned, but would be nice to limit access.

Reply:
Tracked by #1528